src/main/kotlin/org/janelia/saalfeldlab/labels/blocks/n5/LabelBlockLookupFromN5.kt

```diff
@@ -49,10 +56,9 @@ class LabelBlockLookupFromN5(
 						.limit(numIntervals.toLong())
 						.collect(Collectors.toList())
 						.toTypedArray()
-				map.put(id, intervals)
+				map[id] = intervals
 			}
-
-			return map
+			return map.toMap()
```

Comment:
Is it necessary to call `map.toMap()` here?

Reply:
It's not so I went ahead and removed it -- the idea was that the cache would store immutable maps, but Map can't be modified via public interface anyway, so that's fine